### ISSUE TITLE
Does Mapbox.js support IDW heatmaps?

### ISSUE THREAD

**Issue by fullofcaffeine** [mapbox/mapbox.js#1036]
Hi!

Apologies if this is not the right place for this question, so feel free to moderate if needed.

I need to create a vizualization that looks like this one:

https://www.walkscore.com/choice/restaurants/NY/New_York

I've tried using Leaftlet.heat directly, but it doesn't natively support this kind of viz. **More about it here**: Leaflet/Leaflet.heat#30.

I'm wondering - since Mapbox.js builds upon Leaftlet and Leaftlet.heat - if it supports an IDW*[0] heatmap? Somewhat like the one linked above.

Thanks in advance.

*[0] https://en.wikipedia.org/wiki/Inverse_distance_weighting

**Comment by pablocaselas** [comment]
+1

**Comment by fullofcaffeine** [comment]
@pablocaselas It looks like no javascript heatmap lib supports IDW as of now and most of them just assume density of points instead of their values. I've tried modifying heatmap.js with limited success, but I figured out that I could just use a map server and render the heatmap server side, so that the whole zooming and navigation aspect would come for free (by serving the tiles). I'm currently experimenting with OpenGeo and Windshaft, but nothing to show, yet.
